articles/api-management/api-management-faq.md

```diff
@@ -40,6 +40,8 @@ Get the answers to common questions, patterns, and best practices for Azure API
 * [Can I use a self-signed SSL certificate for a back end?](#can-i-use-a-self-signed-ssl-certificate-for-a-back-end)
 * [Why do I get an authentication failure when I try to clone a GIT repository?](#why-do-i-get-an-authentication-failure-when-i-try-to-clone-a-git-repository)
 * [Does API Management work with Azure ExpressRoute?](#does-api-management-work-with-azure-expressroute)
+* [Why do we require a dedicated subnet in Resource Manager style VNET for API Management?](#why-dedicated-subnet-resourcemanger-vnet-apim)
```

Comment:
[Why do we require a dedicated subnet in Resource Manager style VNETs when API Management is deployed into them?